util/containers/Dockerfile

```
@@ -0,0 +1,86 @@
+# syntax=docker/dockerfile:1.4
+
+ARG NODE_VERSION
+ARG RUST_VERSION
+
+
+FROM --platform=${BUILDPLATFORM} docker.io/node:${NODE_VERSION}-alpine AS frontend-build
+
+WORKDIR /build
+COPY frontend/package.json frontend/package-lock.json /build/
+RUN npm i
+
+COPY frontend .
+RUN npm ci
+RUN npx relay-compiler
+RUN npx webpack --progress --mode=production
+
+
+FROM docker.io/rust:${RUST_VERSION}-alpine AS backend-build
+
+ENV CFLAGS    "-fdebug-prefix-map=/root/.cargo/registry/src/github.com-1ecc6299db9ec823/=__dep__"
+ENV RUSTFLAGS "--remap-path-prefix=/build/backend=<src> --remap-path-prefix=/root/.cargo/registry/src/github.com-1ecc6299db9ec823/=<dep>"
+
+WORKDIR /build/backend
+
+RUN apk add --no-cache \
+  make \
+  musl-dev \
+  perl
+
+ARG RUST_TARGET
+RUN rustup target add "${RUST_TARGET}"
+
+COPY backend/Cargo.toml backend/Cargo.lock /build/backend/
+COPY <<EOF /build/backend/src/main.rs
+fn main() {}
+EOF
+RUN cargo build --target "${RUST_TARGET}" --release
+
+COPY --from=frontend-build /build/build /build/frontend/build
+COPY backend .
+ARG GIT_COMMIT_HASH
+RUN cargo build --target "${RUST_TARGET}" --release
+RUN cp "target/${RUST_TARGET}/release/tobira" .
+RUN objcopy --compress-debug-sections tobira
+
+
+FROM docker.io/rust:${RUST_VERSION}-alpine AS rootfs
+RUN mkdir /rootfs \
+ && apk add -p /rootfs --no-cache --initdb \
+ && cp -R /etc/apk/repositories /etc/apk/keys /rootfs/etc/apk/ \
+ && apk add -p /rootfs --no-cache \
+      alpine-baselayout-data \
+      ca-certificates-bundle \
+      musl \
+ && rm -rf \
+      /rootfs/etc/apk \
+      /rootfs/lib/apk \
+      /rootfs/var/cache/apk
+COPY --from=backend-build /build/backend/tobira /rootfs/tobira
+RUN mkdir -p /rootfs/etc/tobira/ \
+ && /rootfs/tobira write-config /rootfs/etc/tobira/config.toml \
+ && sed -i 's/^#address.*/address = "0.0.0.0"/g' /rootfs/etc/tobira/config.toml
```

I'm a bit uncomfortable with this as this will affect any other config values called "address" that we might add in the future. And we would likely not catch it if we don't happen to think of this Dockerfile. I don't really have a better idea right now, but mhhh

---

Thought the same, with no better alternative than to create a static config file for the container image. But that needs updating.

---

This seems to work (i.e. only modifies `http.address`).

```suggestion
 && sed -i '/\[http\]/,/^#address =.*$/ s/^#address =.*$/address = "0.0.0.0"/' /rootfs/etc/tobira/config.toml
```